### ISSUE TITLE
Commands not working

### ISSUE THREAD

**Issue by ValentinPearce** [Shad0wShayd3/Pydex#3]
I have this same issue with all existing keywords except for `search` which is, according to the error message, not defined.
Also typing any undefined prompt or multiple words seem to break the program.
Here is what I get whenI type `help` when prompted:
>Welcome to the Pokedex, what do you need today? (Type help for assistance)
>  `help`
>Traceback (most recent call last):
>  File "pokedex.py", line 15, in <module>
>    if cmd.lower() == helpword() or cmd.lower() == helplet():
>AttributeError: '_Helper' object has no attribute 'lower'

**Comment by Shad0wShayd3** [comment]
This fixed with your pull request, correct? I'd test it now but I'm not at my computer at the moment.

**Comment by ValentinPearce** [comment]
No, the bug I fixed was that pokeloc_en.py couldn't be opened because of an `\xc3` character. It's a bug I had on one of my projects and the origin is the encoding.

**Comment by Shad0wShayd3** [comment]
Ah. I'll look into this then. If I had to guess, I'd say pokeloc_en is probably causing this. Might just be a problem with the names. Probably try moving the localization stuff to json first,now that the dex entries are done.

**Comment by Shad0wShayd3** [comment]
I updated pokefunc.py and pokedex.py to remove pokeloc, does this fix your problem? I am unable to replicate the issue.

**Comment by ValentinPearce** [comment]
Unfortunately it doesn't... What version of python are you using?
My python3 doesn't see the pykemon module and I get the bug with my python2.7

**Comment by Shad0wShayd3** [comment]
I'm using Python 3.4.2, and the modded pykemon files I have in this repo.
